docs/installation.md

```diff
@@ -20,6 +20,12 @@ Please see [Getting Started with Kubeflow](https://www.kubeflow.org/docs/started
 
 Run the script `hack/install.sh`, then the specification of the kernel will be installed to `${HOME}/.local/share/jupyter/kernels/kubeflow`, then Jupyter will know the information about Ciao.
 
+Then we need to create a configuration file `$HOME/.ciao.yaml`:
```

Comment:
nit: use `~/.ciao/config.yaml` in case we need more configs in the future.

Reply:
SGTM, I will update it.